### DISCUSSION TITLE
Invoking a bash script on opening Geany?

### DISCUSSION THREAD

**Discussion by ineuw** [geany/geany#3538] [Q&A]
I am a new user, and wonder if gaining focus can be triggered from within Geany on opening with an option perhaps, rather than using a bash script to start the app, followed by ` "xdotool click 1" ` to insure focus.

**Comment by elextr** [comment]
No, at best Geany can only __request__ that the window manager give it focus, which it does, it issues a GTK grab focus on the edit pane, but the window manager does not have to honour that, and depending on the manager rules many don't, especially on Wayland based systems.  Xdotool is an X11 tool, and also may not work on Wayland systems.
